src/main/java/org/galatea/starter/entrypoint/IexRestController.java

```diff
@@ -1,5 +1,11 @@
 package org.galatea.starter.entrypoint;
 
+import java.time.LocalDate;
```

Comment:
Avoid imports you don't use

Reply:
I forgot to get rid of those!

Reply:
I think I'm using it! IntelliJ seems to delete any imports I'm not using automatically.